base/special/trig.jl

```diff
@@ -106,7 +106,7 @@ Compute ``\\sin(\\pi x)`` more accurately than `sin(pi*x)`, especially for large
 function sinpi(x::T) where T<:AbstractFloat
     if !isfinite(x)
         isnan(x) && return x
-        throw(DomainError())
+        throw(DomainError(x, "`x` cannot be infinite."))
```

Comment:
Slight inconsistency between errors in this file? ("`x` cannot be infinite" versus "`x` must be finite"?)